site/pages/cookies.page.tsx

```diff
@@ -139,7 +137,7 @@ export const getServerSideProps = (ctx: NextPageContext): { props: CookiePrefere
 
     const trackingDefaultValue = tracking ? "on" : "off";
 
-    return { props: { settingsSaved, trackingDefaultValue, csrfToken } };
```

Comment:
How come this page doesnt need it?

Reply:
The token is injected in _app.page.tsx so it doesn't need to be passed through in the getServerSideProps